openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -0,0 +1,316 @@
+            <div class="ui-staff ui-toggle-visibility is--collapsed">
+                <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+                    <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+                    <span>{% trans "Peer Assessments for This Learner" %}</span>
+                </h2>
+                <div class="ui-toggle-visibility__content">
+                    {% if peer_assessments %}
+                        {% for assessment in peer_assessments %}
+                            {% with peer_num=forloop.counter %}
+                                <h4 class="title--sub"> {% trans "Peer" %} {{ peer_num }}: </h4>
+                                <table class="staff-info__status__table" summary="{% trans "Assessment" %}">
+                                    <thead>
+                                    <tr>
+                                        <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
+                                        <th abbr="Selected Option" scope="col">{% trans "Selected Option" %}</th>
+                                        <th abbr="Points" scope="col">{% trans "Points" %}</th>
+                                        <th abbr="Points Possible" scope="col">{% trans "Points Possible" %}</th>
+                                    </tr>
+                                    </thead>
+
+                                    <tbody>
+                                    {% for criterion in rubric_criteria %}
+                                        {% for part in self_assessment.parts %}
+                                            {% if part.option.criterion.name == criterion.name %}
+                                                <tr>
+                                                    <td class="label">{{ criterion.label }}</td>
+                                                    <td class="value">{{ part.option.label }}</td>
+                                                    <td class="value">{{ part.option.points }}</td>
+                                                    <td class="value">{{ criterion.total_value }}</td>
+                                                </tr>
+                                            {% endif %}
+                                        {% endfor %}
+                                    {% endfor %}
+                                    </tbody>
+                                </table>
+                                <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
+                                <div class="student__answer__display__content">
+                                    {{ assessment.feedback|linebreaks }}
+                                </div>
+                            {% endwith %}
+                        {%  endfor %}
+                    {% else %}
+                        <p>{% trans "This learner currently has no peer assessments." %}</p>
```

Comment:
@catong @explorerleslie I modified this section so that it always displays if the problem is configured to support peer assessments. Before this change, it meant that the section would not be shown, so there was no way for staff to tell that this problem supports peer assessments but just happens not to have any.

@catong Let me know if you think the language for this is clear.